Tests/Simulation/SimulatedLoadTests.cs

```diff
@@ -81,6 +82,25 @@ public async Task Single_ABP_Simulated_Device()
             AssertMessageAcknowledgement(device, messageCount);
         }
 
+        [Fact]
+        public async Task Ensures_Disconnect_Happens_For_Losing_Gateway_When_Connection_Switches()
+        {
+            // arrange
+            var device = new SimulatedDevice(TestFixtureSim.Device1003_Simulated_ABP, simulatedBasicsStation: new[] { this.simulatedBasicsStations.First() }, logger: this.logger);
+            await SendConfirmedUpstreamMessages(device, 1);
+
```

Comment:
Let's wait 15 seconds between the two confirmed messages and log something that is helping understanding what's happening there

Reply:
In addition, let's send at least 3 messages to both gateways instead of one and let's verify that the connection does not switch before than what we expect

Reply:
Done in 46a820a8afc9985f807bd2dae09074846f5218a4